CONTRIBUTING.md

```diff
@@ -12,13 +12,12 @@ please consider contributing in the following areas:
 
 ## Development Setup
 
-To get started hacking on pdoc, please install a recent version of Python (we recommend at least Python 3.9). Then, do
```

Comment:
I don't know if uv takes care of also installing the correct python version. I tried it before, and it picked 3.9, which causes this error:

```
lint: commands[1]> mypy pdoc test
test/testdata/misc_py312.py:13: error: invalid syntax  [syntax]
Found 1 error in 1 file (errors prevented further checking)
lint: exit 2 (12.30 seconds) /Users/feuh/repos/pdoc> mypy pdoc test pid=95808
lint: FAIL ✖ in 15.63 seconds
```

Reply:
Should be fixed now, thanks!